internal/provider/exception_resource.go

```diff
@@ -47,6 +47,7 @@ type exceptionResourceModel struct {
 	Action            types.String `tfsdk:"action"`
 	CheckMrns         types.List   `tfsdk:"check_mrns"`
 	VulnerabilityMrns types.List   `tfsdk:"vulnerability_mrns"`
+	ExceptionId       types.String `tfsdk:"exception_id"`
```

Comment:
what happens to existing state files where this doesnt exist

Reply:
hmm. good question. ill check

Reply:
ok. i tested this all out.
- if a user has an exception in their state file and there are no changes between the rest of the data in the exception and the state file, the tf wont complain because the exception id is optional
- if a user has an exception in their state file and there are changes between what exists and the state file, then the tf will trigger an update, which will now search for the exception, if found delete and recreate, if not found return error and instruct to import

Reply:
working on import resource now

Reply:
import resource should be working now as well